### DISCUSSION TITLE
remove discussion

### DISCUSSION THREAD

**Discussion by RikStarmans** [alan-turing-institute/sktime#3766] [Q&A]
I can create but not remove dicussion on Github.
Seems not possible
https://github.com/community/community/discussions/2948

**Comment by fkiraly** [comment]
I can delete discussions, but I think only maintainers can do that?

**Reply by fkiraly** [reply]
which discussion do you want deleted and why?

General GitHub policy is to keep historical discussions (for a scientific audit trail), except for where content is illegal or privacy violating or the like.
